index.d.ts

```diff
@@ -0,0 +1,55 @@
+type RecordObject = import('cql-execution').RecordObject;
+type PatientObject = import('cql-execution').PatientObject;
+type DataProvider = import('cql-execution').DataProvider;
+type RetrieveDetails = import('cql-execution').RetrieveDetails;
+type Code = import('cql-execution').Code;
+type Date = import('cql-execution').Date;
+type Interval = import('cql-execution').Interval;
+
+class FHIRObject implements RecordObject {
+  constructor(json: any, typeInfo: any, modelInfo: any);
+  get(field: string): any;
+  getId(): string;
+  getCode(field: string): Code;
+  getDate(field: string): Date;
+  getInterval(field: string): Interval;
+  getDateOrInterval(field: string): Date | Interval;
+  _is(typeSpecifier: any): boolean;
+  _typeHierarchy(): any;
+  getTypeInfo(): any;
+}
+
+class Patient extends FHIRObject implements PatientObject {
+  constructor(bundle: any, modelInfo: any, patientSourceOptions: PatientSourceOptions);
+  findRecords(profile: string, retrieveDetails?: RetrieveDetails): FHIRObject;
```

Comment:
This is very similar to [PatientObject.findRecords](https://github.com/cqframework/cql-execution/blob/1c6fd23ccc80daedf9f2e6f9de51fffadf1b19de/src/types/cql-patient.interfaces.ts#L43), but I assume you redeclare it so you can provide the more specific return type.

That said, this returns a list, so the return type should be `FHIRObject[]`.
```suggestion
  findRecords(profile: string, retrieveDetails?: RetrieveDetails): FHIRObject[];
```